apiserver/facades/client/action/action.go

```diff
@@ -190,7 +190,10 @@ func (a *ActionAPI) Enqueue(arg params.Actions) (params.ActionResults, error) {
 		currentResult := &response.Results[i]
 		receiver, err := tagToActionReceiver(action.Receiver)
 		if err != nil {
-			currentResult.Error = common.ServerError(err)
+			currentResult.Error = &params.Error{
```

Comment:
This change isn't needed - common.ServerError() will construct the correct params.Error according to the err passed in.